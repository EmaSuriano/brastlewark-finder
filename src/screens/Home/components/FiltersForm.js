import styled from 'styled-components';

const FiltersForm = styled.form`
  display: flex;
  align-items: center;
  height: auto;
  margin: 1em 0;
`;

export default FiltersForm;
