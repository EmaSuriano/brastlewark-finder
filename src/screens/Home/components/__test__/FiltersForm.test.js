import React from 'react';
import { shallow } from 'enzyme';
import FiltersForm from '../FiltersForm';

describe('<FiltersForm />', () => {
  it('render', () => {
    expect(shallow(<FiltersForm />)).toMatchSnapshot();
  });
});
