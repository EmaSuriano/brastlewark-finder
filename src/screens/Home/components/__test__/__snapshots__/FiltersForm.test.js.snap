// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FiltersForm /> render 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  height: auto;
  margin: 1em 0;
}

<form
  className="c0"
/>
`;
